syntax = "proto2";

package planning;
import "speed_profile_conf.proto";
import "path_planner_conf.proto";
import "environment_conf.proto";

message PlanningConf {
    optional EnvironmentConf environment_conf = 1;
    optional PathPlannerConf path_planner_conf = 2;
    optional speed_profile.SpeedProfileConf speed_profile_conf = 3;
    optional string path_planner_conf_path = 4;
    optional string speed_profile_conf_path = 5;
    optional string environment_conf_path = 6;
    optional bool run_path = 7;
    optional bool run_speed = 8;
}
